win32/select.c

```diff
@@ -87,7 +87,7 @@ PHPAPI int php_select(int max_fd, fd_set *rfds, fd_set *wfds, fd_set *efds, stru
 
 	if (n_handles == 0) {
 		/* plain sockets only - let winsock handle the whole thing */
-		return select(max_fd, rfds, wfds, efds, tv);
+		return select(NULL, rfds, wfds, efds, tv);
```

Comment:
Yeah, it can be ignored, doesn't look like a change required. Btw nfds is not a pointer ;)